postgres/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.4.1](https://github.com/cap-js/cds-dbs/compare/postgres-v1.4.0...postgres-v1.4.1) tbd
+
+### Fixed
+
+- `cds build` now generates the correct output folder structure for Node.js and Java apps.
+
```

Comment:
please no manual changelog entries, if youre following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) it will be automatically done for you